CONTRIBUTING.md

```diff
@@ -0,0 +1,97 @@
+# Welcome contributors to Alva
+💓 First of all, we'd love for you to contribute to our source code. 💓
+
+## Code of Conduct
+We belive in a welcoming and inclusive environment / community, that's why we enforce our Code of Conduct.
+For more information -> [Contributor Covenant Code of Conduct](CODE_OF_CONDUCT.md).
+
+
+## Table of Contents
+- [Link collection](#short-collection-of-links-to-important-resources)
+- [Setup for contributers](#setup-for-contributers)
+- [Issues](#issues)
+	- [Good first issue for Contributors](#good-first-issue)
+	- [Lable structur](#labels)
+	- [Working on issues](#working-on-issues)
+- [Style Guide / Coding conventions](#style-guide-/-coding-conventions)
+	- [Coding conventions & guidelines](#coding-conventions-&-guidelines)
+
+
+## Short collection of Links to Important Resources
+   * bugs / feature requests: https://github.com/meetalva/alva/issues
+   * website: https://meetalva.io
+   * sprint board: https://github.com/meetalva/alva/projects/2
+   * backlog board: https://github.com/meetalva/alva/projects/3
+
+
+## Setup for contributers
+To add a contribution to Alva (improve its features, fix issues),
+get the source of both the application itself,
+and the kickstart styleguide (designkit):
+
+* Clone this repository [alva](https://github.com/meetalva/alva): git@github.com:meetalva/alva.git
+* Clone [designkit](https://github.com/meetalva/designkit) repository: git@github.com:meetalva/designkit.git
+* Run in the designkit repository:
+```shell
+npm i && npm run build
+```
+* Run in the main (Alva) respository:
+```shell
+npm i && npm start
+```
+
+
+## Issues
+### Good first issue
+A good first issues for contribution could be
+* reporting a bug
+* create a feature request
+* work on an issue with the label *good first issue*
+
+### Labels
+- *good first issue* - if you are new to *Alva* or open source this could be a good first issue
+- *help wanted* - issue where we need help
+- *type: bug* - issue for an error/ failure of Alva
+- *type: feature* - issue for feature request
+- *type: design* - issue for a design todo
+- *type: documentation* - issue for needed, missing or unclear documentation
+- *status: in progress* - issue is currently in progress
+- *status: has PR* - issue has already an open pull request
+- *priority: low* - issue which is a nice to have, like a backlog issue
+- *priority: mid* - issue needed in the near future, but not super urgent
+- *priority: high* - issue must be resolved as soon as possible
+
+### Working on issues
+Issues should be provided as branches starting with "feat/", as pull requests.
```

Comment:
Would it make more sense to write something like this:
"Pull requests should be provided as branches starting with 'feat/'"

Reply:
as below - only copied this from readme.

Yes, it's more clear.

Also, I would prefer to say - branches should start similar to commit conventions: feat, docs, bug, chore etc.
BUT I don't know what you as project team decided :)

Reply:
I thing its a good idea. But i also have the feeling that the name of a branch is not that important. After the merge no one will look at the name again :)